addons/website_slides/static/src/js/slides.js

```diff
@@ -115,8 +115,9 @@ publicWidget.registry.websiteSlidesEmbed = publicWidget.Widget.extend({
     _onIframeViewerReady: function (ev) {
         // TODO : make it work. For now, once the iframe is loaded, the value of #page_count is
         // still now set (the pdf is still loading)
-        var $iframe = $(ev.currentTarget);
-        var maxPage = $iframe.contents().find('#page_count').val();
+        var maxPage = parseInt(
+            $('iframe.o_wslides_iframe_viewer').contents().find('#page_count').text()
+        );
```

Comment:
I think we could indeed move this to the onchange above, as it only matters when we actually want to change the page.
```js
if (!this.maxPage) {
   this.maxPage = parseInt(
        $('iframe.o_wslides_iframe_viewer').contents().find('#page_count').text()
    );
    // then perhaps set this max to the `<input>`?
}
```
We don't need more for a quick fix. (Keeping this handler with the todo for improving on it later if necessary,  but simplifying to `new SlideSocialEmbed(this, 0)`?)